server/lib/report_server_web/live/new_report_live/form.html.heex

```diff
@@ -73,16 +73,28 @@
         <.button :if={!blank?(@form.params[filter]) && has_more_filters && last_filter} type="button" phx-click="add_filter"><.icon name="hero-funnel" /> Add Filter</.button>
       </div>
     </div>
-    <div class="mt-4 flex items-center gap-4">
+
+    <%= if !blank?(@form.params["filter1"]) do %>
+
+      <div class="mt-4 flex items-center gap-4">
         <span class="font-bold">Include last run dates:</span>
         <label for="start_date" class="whitespace-nowrap">Earliest date:</label>
         <.input type="date" id="start_date" field={@form["start_date"]} />
         <label for="end_date" class="whitespace-nowrap">Latest date:</label>
         <.input type="date" id="end_date" field={@form["end_date"]} />
-    </div>
-    <div class="mt-4">
-      <.button :if={!blank?(@form.params["filter1"])} type="button" phx-click="submit_form"><.icon name="hero-bolt" /> Run Report</.button>
-    </div>
+      </div>
+
+      <div :if={@form_options.enable_hide_names}>
+        <div class="flex gap-2 items-center">
+          <.input type="checkbox" id="hide_names" field={@form["hide_names"]} />
+          <label for="hide_names" class="whitespace-nowrap">Hide names</label>
```

Comment:
For improved privacy, I'd suggest we hide names by default for everyone, and admins have to check a 'Show names' option to see them.

Reply:
Probably a good idea - we an update that in a future PR if Kiley agrees.